src/toil/utils/toilLaunchCluster.py

```diff
@@ -139,26 +140,33 @@ def main():
 
     if not config.zone:
         raise RuntimeError('Please provide a value for --zone or set a default in the TOIL_' +
-                           config.provisioner.upper() + '_ZONE enviroment variable.')
+                           config.provisioner.upper() + '_ZONE environment variable.')
 
     cluster = clusterFactory(provisioner=config.provisioner,
                              clusterName=config.clusterName,
                              zone=config.zone,
                              nodeStorage=config.nodeStorage)
-
-    cluster.launchCluster(leaderNodeType=config.leaderNodeType,
-                          leaderStorage=config.leaderStorage,
-                          owner=owner,
-                          keyName=config.keyPairName,
-                          botoPath=config.botoPath,
-                          userTags=tagsDict,
-                          vpcSubnet=config.vpcSubnet,
-                          publicKeyFile=config.publicKeyFile,
-                          azureStorageCredentials=config.azureStorageCredentials)
+    try:
```

Comment:
I think this might be a bad place for this.  If you're using this call for status, it should be purely inside of it's own function.  That way you separate the two intentions, getting status, and actually expecting to spin up a cluster.